modules/nf-core/wipertools/fastqscatter/main.nf

```diff
@@ -0,0 +1,55 @@
+process WIPERTOOLS_FASTQSCATTER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+    val(num_splits)
+
+    output:
+    tuple val(meta), path("${out_folder}/*") , emit: chunks
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args ?: ''
+    def prefix      = task.ext.prefix ?: "${meta.id}"
+    def args_list   = args.tokenize()
+    out_folder      = (args_list.contains('--out_folder') ? args_list[args_list.indexOf('--out_folder')+1] : (args_list.contains('-o') ? args_list[args_list.indexOf('-o')+1] : 'chunks'))
```

Comment:
Can you format this, I can't tell what it is doing with the nested conditions.

Reply:
OK

Reply:
not sure it is properly formatted now